app/ui/page.tsx

```diff
@@ -253,6 +255,49 @@ const TagColorSchemes = [
   "info",
 ];
 
+interface IconListProps {
+  name: IconName;
+  disabled?: boolean;
+  active?: boolean;
+  variant?: ButtonVariantProp;
+  size?: ButtonSizeProp;
+}
+
+const iconButtonList: Array<Array<IconListProps>> = [
+  [
+    { name: "add-fill", size: "xs" },
+    { name: "add-fill", disabled: true, size: "xs" },
+    { name: "add-fill", active: true, size: "xs" },
+    { name: "add-fill", variant: "pastel", size: "xs" },
+    { name: "add-fill", variant: "outline", size: "xs" },
+    { name: "add-fill", variant: "ghost", size: "xs" },
+  ],
+  [
+    { name: "add-fill", size: "sm" },
+    { name: "add-fill", disabled: true, size: "sm" },
+    { name: "add-fill", active: true, size: "sm" },
+    { name: "add-fill", variant: "pastel", size: "sm" },
+    { name: "add-fill", variant: "outline", size: "sm" },
+    { name: "add-fill", variant: "ghost", size: "sm" },
```

Comment:
it might be more interesting if we do each array with a diff icon, like settings wheel etc, would display several icons instead of just "add-fill"

Reply:
IMHO as long as the same icon is presented in a single row (to check the changes for the same icon and different sizes and variants), I'm ok with this

Reply:
Yeah, we should keep the same per row.

Reply:
Will leave it as it is for now